data-transfer-object/src/main/java/com/iluwatar/data/transfer/object/OrderDto.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * The MIT License
+ * Copyright (c) 2014 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.iluwatar.data.transfer.object;
+
+/**
+ * This example is of online phone orders. We have products, customers and their orders.
+ * Products, customers and orders data are saved in their respective json files. 
+ * <p>
+ * products.json is an array of product object. Product object contains product name, price, etc..
+ * <p>
+ * customers.json is an array of customer object. Customer object contains customer name, address, etc..
+ * <p>
+ * Orders.json is an array of order object. Order contains the unique identifier of the customer who 
+ * placed the order and unique identifier of the product he/she ordered.
+ * <p>
+ * I considered the index of product in products json array as its unique identifier for simplicity in coding. 
```

Comment:
> I considered the index of product in products json array as its unique identifier for simplicity in coding.

Can be rephrased as "For simplicity the indexes of Customers, Products and Orders are considered to be their unique identifiers."